lib/linebuffer.js

```diff
@@ -24,6 +24,31 @@ var dccount = 0;
 var httperr;
 var queue;
 
+// check response to see need for buffering into queue
+const checkResponse = (error, response, body) => {
+    if (error) {
+        debug('req error ' + error.code + ': ' + error);
+        if (!httperr) { httperr = Date.now(); }
```

Comment:
This precedes you so you don't have to change it but as a reader, this is pretty mystifying on a couple levels - what the var means and why it's being set to a date.

Reply:
renaming makes it easier to understand why it's set to datetime object